workflow/scripts/h5.jl

```diff
@@ -0,0 +1,188 @@
+function getiftversion()
+    deps = Pkg.dependencies()
+    iftversion = []
+    for (_, dep) in deps
+        dep.is_direct_dep || continue
+        dep.version === nothing && continue
```

Comment:
They actually say people should use `===` instead of `==` when comparing against `nothing`. Apparently it's faster because `==` does a type conversion before doing the comparison.